pools/custom_pools_test.go

```diff
@@ -0,0 +1,20 @@
+package pools
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	customValidatorFile = "pubkeys.json"
```

Comment:
great! if possible, i think its better to avoid having files. imho the file should be created in the test and then removed.